src/feed-input.scss

```diff
@@ -0,0 +1,78 @@
+@import "./new-settings";
+@import "./mixins";
+
+$fd-feed-input-row-height: 3rem;
+$fd-feed-input-row-max-height: 20rem;
+$fd-feed-max-width: calc(100vw * 2 / 3);
+
+$block: #{$fd-namespace}-feed-input;
+$thumb: #{$block}__thumb;
+$textarea: #{$block}__textarea;
+$submit-btn: #{$block}__submit-button;
+
+.#{$block} {
+  @include fd-reset();
+
+  display: flex;
+  flex-direction: row;
+  padding: 1rem 0.5rem 1rem 1rem;
+  background-color: #fff;
+  background-color: var(--sapField_Background, #fff);
+
+  @include fd-screen(m) {
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+  }
+
+  @include fd-screen(l) {
+    max-width: $fd-feed-max-width;
+  }
+
+  .#{$thumb} {
```

Comment:
We are following BEM. In this case it's better to use 
```&__thumb {}```

Reply:
`$thumb: #{$block}__thumb;` is not it the same?
I put it in variable as it's used several times

Reply:
It is the same but it goes against the naming conventions we adopted in the project